recipes/_download_artifacts.rb

```diff
@@ -26,6 +30,16 @@
   retries 2
 end
 
+remote_file tmp_admin_zip_path do
+  source "http://eu.dl.alfresco.com.s3.amazonaws.com/release/enterprise/process-services-1.6/#{aps_admin_version}/activiti-admin-#{aps_admin_version}.zip"
```

Comment:
would it change the name of the folder `process-services-1.6` if version changes?
If so that 1.6 shouldn't it be parametrized as well?

Reply:
Probably.